.env.example

```
@@ -71,11 +92,10 @@ WEBDRIVER_TIMEOUT_MS="60000"
 WEB_RENDER_RETRIES="2"
 
 ########################################
-# Consensus Rollup
-# Hardhat port
-HARDHAT_URL='http://hardhat'
+# Optional legacy Hardhat inspection bridge. Studio protocol state and ghost
+# addresses are DB-authoritative and do not require this service.
+HARDHAT_URL=''
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)